cmd/plugin/plugin.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 D2iQ, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package plugin
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/mesosphere/dkp-cli-runtime/core/output"
+)
+
+func NewCommand(out output.Output) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "plugin",
+		Short: "Plugin",
+	}
+
+	// Subcommands
+	cmd.AddCommand(InstallCommand(out))
+	cmd.AddCommand(AddCommand(out))
+	cmd.AddCommand(RemoveCommand(out))
+	cmd.AddCommand(ListCommand(out))
+
+	return cmd
+}
+
+// InstallCommand creates a new command to install
+func InstallCommand(out output.Output) *cobra.Command {
+	return &cobra.Command{
+		Use:   "install",
+		Short: "Installs a tool for a specific plugin",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			out.V(6).Info(fmt.Sprintf("args: %v", args))
+			return nil
+		},
+	}
+}
+
+// AddCommand creates a new command to add a plugin
+func AddCommand(out output.Output) *cobra.Command {
+	return &cobra.Command{
+		Use:   "add",
+		Short: "Adds a plugin",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			out.V(6).Info(fmt.Sprintf("args: %v", args))
+			return nil
+		},
+	}
+}
```

Comment:
[question] do we need this one as well, what is the difference between this and `install`

Reply:
Depends, this is for adding a plugin to the source and the install is to install a specific version of that plugins tool.
I'm not sure we can manage this by only one command, i.e. `install go 1.19` would need to install the go plugin to asdf in the current setup, but there are a lot of plugins out there and we would need to hardcode all of them

Reply:
Do you think one command is enough for now?